diff --git a/lib/coreBundle.js b/lib/coreBundle.js
index 7d38afcb713a4ca31526e9edff5ef044487260a9..5c6cbf4cc5fc68f5ba8d73bb51162cb9b8ada002 100644
--- a/lib/coreBundle.js
+++ b/lib/coreBundle.js
@@ -39151,7 +39151,7 @@ var init_crPage = __esm({
         this.updateOffline();
         this.updateExtraHTTPHeaders();
         this.updateHttpCredentials();
-        this._networkManager.setRequestInterception(true);
+        this.updateRequestInterception();
         this.initScriptTag = import_crypto16.default.randomBytes(20).toString("hex");
         this._mainFrameSession = new FrameSession(this, client, targetId, null);
         this._sessions.set(targetId, this._mainFrameSession);
